src/components/CalendarMonthGrid.jsx

```diff
@@ -95,11 +115,22 @@ export default class CalendarMonthGrid extends React.Component {
     this.props.onMonthTransitionEnd();
   }
 
+  getMonths(initialMonth, numberOfMonths) {
+    let month = initialMonth.clone().subtract(1, 'month');
```

Comment:
As this function is written, it is a little difficult to grok. To avoid doing mental acrobatics, what do you think about moving this into your loop? It shouldn't be too hard if methods like `add` or `subtract` can take negative numbers.